pkg/config/config.go

```diff
@@ -230,6 +235,9 @@ var defaultLintersSettings = LintersSettings{
 	Gocritic: GocriticSettings{
 		SettingsPerCheck: map[string]GocriticCheckSettings{},
 	},
+	Godox: GodoxSettings{
+		Keywords: []string{"FIXME", "TODO"},
```

Comment:
it looks like default values are duplicated in two places: here and in [godox repo](https://github.com/matoous/godox/blob/master/godox.go#L14). And they differ :) Maybe remove defaults here and leave only defaults in godox?

Reply:
Removed.